### PR TITLE
Interop - Update AAP to test on version 2.5 on OCP 4.18

### PR DESCRIPTION
**Pull request by oharan2**

### CODE REVIEW

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-ansible-tower-qa-devel-ocp4.18-lp-interop-aap-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-ansible-tower-qa-devel-ocp4.18-lp-interop-aap-interop-aws-fips

File: ci-operator/config/ansible/tower-qa/ansible-tower-qa-devel__ocp4.18-lp-interop.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ resources:
       memory: 200Mi
 tests:
 - as: aap-interop-aws
-  cron: 0 0 11 6 *
+  cron: 0 20 31 2 *
```
Comment (dfrazzette) [inline ci-operator/config/ansible/tower-qa/ansible-tower-qa-devel__ocp4.18-lp-interop.yaml:25]:
```suggestion
  cron: 0 23 31 2 *
```

File: ci-operator/config/ansible/tower-qa/ansible-tower-qa-devel__ocp4.18-lp-interop.yaml [hunk 2]
```diff
@@ -51,7 +51,7 @@ tests:
     - ref: aap-tests-integration
     workflow: firewatch-ipi-aws
 - as: aap-interop-aws-fips
-  cron: 0 0 11 6 *
+  cron: 0 20 31 2 *
```
Comment (dfrazzette) [inline ci-operator/config/ansible/tower-qa/ansible-tower-qa-devel__ocp4.18-lp-interop.yaml:54]:
```suggestion
  cron: 0 23 31 2 *
```

File: ci-operator/jobs/ansible/tower-qa/ansible-tower-qa-devel-periodics.yaml [hunk 3]
```diff
@@ -519,7 +519,7 @@ periodics:
         secretName: result-aggregator
 - agent: kubernetes
   cluster: build03
-  cron: 0 0 11 6 *
+  cron: 0 20 31 2 *
```
Comment (dfrazzette) [inline ci-operator/jobs/ansible/tower-qa/ansible-tower-qa-devel-periodics.yaml:522]:
```suggestion
  cron: 0 23 31 2 *
```

**Pull request merged**